kuma/wiki/tests/test_views.py

```diff
@@ -2281,48 +2282,180 @@ def test_quick_review(self):
             eq_(data_dict['expected_tags'], review_tags)
 
     @pytest.mark.midair
-    def test_edit_midair_collision(self):
+    def test_edit_midair_collisions(self):
+
+        def midair_collision_test(is_ajax):
```

Comment:
The way we've done this for other cases is to define a function that takes a optional parameter, like:

```
def test_edit_midair_collision(self, is_ajax=False):
    # Code that works both ways
```

and then, have a second test that calls it with the alternate option:

```
def test_edit_midair_collision_ajax(self):
    self.test_edit_midair_collision(is_ajax=True)
```

Please replace the inner function with this alternate pattern.